bootloader/bootloader.go

```diff
@@ -234,6 +235,19 @@ func InstallBootConfig(gadgetDir, rootDir string, opts *Options) error {
 	return fmt.Errorf("cannot find boot config in %q", gadgetDir)
 }
 
+type bootloaderNewFunc func(rootdir string, opts *Options) Bootloader
+
+var (
+	//  bootloaders list all possible bootloaders by their constructor
+	//  function.
+	bootloaders = []bootloaderNewFunc{
+		newGrub,
+		newUboot,
```

Comment:
/me notes that this is note the same order as the original code, is this an issue? I don't think so, but just thought I would call it out

Reply:
I've pushed a patch that preserves the original order of uboot -> grub -> android -> lk.